3.component.md

```diff
@@ -73,10 +73,10 @@ metadata:
   annotations:
     version: v1.0.0
     description: >
-      Sample component that describes the administrative interface for our Twitter bot.
+      Sample component that describes a very cool workload.
 spec:
-  type: Server
-  settings:
+  type: Server # <--- the type of the workload
+  settings: # <---- the settings of the workload
```

Comment:
the `fieldPaths` of `parameters` used in the sample may be revised according to new spec field structure.
> "spec.containers[0].image" ==> "spec.settings.containers[0].image"

Reply:
Good catch, just fixed.